src/components/Video/PreviewControls/PreviewControls.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useRef, useEffect } from 'react';
+import styles from './PreviewControls.module.scss';
+import isTouchDevice from '../../../utils/isTouchDevice';
+import useMousePosition from '../../../utils/hooks/useMousePosition';
+
+const PreviewControls = ({ handleClick }: { handleClick: () => void }) => {
+  const isMobile = isTouchDevice();
+
+  const parentRef = useRef<HTMLDivElement>();
+  const followTextRef = useRef<HTMLSpanElement>();
+
+  const mousePosition = useMousePosition(parentRef.current);
```

Comment:
With something that updates as often as this, I would steer clear of using reactivity. Makes more sense to directly hook in to event handler in an effect, and update the position directly that way.

Reply:
Yea good shout tbh, overcomplicated it. Removed the hook completely